robottelo/constants.py

```diff
@@ -326,22 +326,16 @@
         'configuration',
         'create_arf_reports',
         'create_containers',
-        'create_job_invocations',
-        'create_job_templates',
```

Comment:
I removed these (and some more) as they were duplicate (their duplicates can be found in the appropriate sections below in the list)